src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -97,8 +97,18 @@ internal NamedTypeSymbol GetWellKnownType(WellKnownType type)
                 }
                 else
                 {
-                    result = this.Assembly.GetTypeByMetadataName(
-                        mdName, includeReferences: true, useCLSCompliantNameArityEncoding: true, isWellKnownType: true, warnings: warnings);
+                    if (type <= WellKnownType.CSharp7Sentinel)
```

Comment:
Duplicating the line means the reader has to spot the differences between the two and also increases the chance the two cases will get out of sync.